TVC_MVP/County.swift

```diff
@@ -12,5 +12,20 @@ struct County {
 
     let countyID: Int
     let countyName: String
+    var countyExtras: CountyExtras?
+
+    init(countyID: Int, countyName: String) {
+        self.countyID = countyID
+        self.countyName = countyName
+    }
     
 }
+
+struct CountyExtras {
+
+    var population: Int?
+    var newBorns: Int?
+
+    init() { }
```

Comment:
A better approach for the same behaviour would be to have the parameters in the `init` with a default value.